<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="HP" failures="0" tests="2" name="com.crm.practice.ContactAndOraganizationTest" time="0.023" errors="1" timestamp="2022-04-07T07:45:03 IST" skipped="1">
  <testcase classname="com.crm.practice.ContactAndOraganizationTest" name="ContactAndOraganizationTest" time="0.000">
    <skipped/>
  </testcase> <!-- ContactAndOraganizationTest -->
  <system-out/>
  <testcase classname="com.crm.practice.ContactAndOraganizationTest" name="openDatabase" time="0.023">
    <error message="Unresolved compilation problems: 
	The type java.sql.Driver cannot be resolved. It is indirectly referenced from required .class files
	The method registerDriver(Driver) from the type DriverManager refers to the missing type Driver
" type="java.lang.Error">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
The type java.sql.Driver cannot be resolved. It is indirectly referenced from required .class files
The method registerDriver(Driver) from the type DriverManager refers to the missing type Driver

at com.crm.genericUtility.DatabaseUtility.getMysqlDatabaseConnection(DatabaseUtility.java:36)
at com.crm.genericUtility.BaseClass.openDatabase(BaseClass.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:317)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- openDatabase -->
  <system-out/>
</testsuite> <!-- com.crm.practice.ContactAndOraganizationTest -->
